internal/backends/gl/lib.rs

```diff
@@ -16,18 +16,13 @@ use glwindow::*;
 mod glcontext;
 use glcontext::*;
 pub(crate) mod event_loop;
-mod images;
+mod renderer;
```

Comment:
```rust
mod renderer { mod femtovg; }
```

That way there is no need for a file with just other mod in it.
As you prefer

Reply:
Good idea :)